src/Banner.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import Box from './Box'
+import Flex from './Flex'
+import Text from './Text'
+import Icon from './Icon'
+import CloseButton from './CloseButton'
+import Heading from './Heading'
+import PropTypes from 'prop-types'
+
+const bannerColors = {
+  green: {
+    backgroundColor: 'green',
+    color: 'white',
+    icon: 'success'
+  },
+  lightGreen: {
+    backgroundColor: 'lightGreen',
+    color: 'text',
+    icon: 'success'
+  },
+  red: {
+    backgroundColor: 'red',
+    color: 'white',
+    icon: 'warning'
+  },
+  lightRed: {
+    backgroundColor: 'lightRed',
+    color: 'text',
+    icon: 'warning'
+  },
+  orange: {
+    backgroundColor: 'orange',
+    color: 'white',
+    icon: 'attention'
+  },
+  lightOrange: {
+    backgroundColor: 'lightOrange',
+    color: 'text',
+    icon: 'attention'
+  },
+  blue: {
+    backgroundColor: 'blue',
+    color: 'white',
+    icon: 'information'
+  },
+  lightBlue: {
+    backgroundColor: 'lightBlue',
+    color: 'text',
+    icon: 'information'
+  }
+}
+
+const Banner = props => {
+  const bannerColor = bannerColors[props.bg] || {}
+  const icon = props.iconName || bannerColor.icon
+
+  return (
+    <Box {...props} bg={bannerColor.backgroundColor} color={bannerColor.color}>
+      <Flex justify="space-between" align="flex-start">
+        {!!icon &&
+          !!props.showIcon && <Icon name={icon} mr={3} size={28} mt="-2px" />}
```

Comment:
Is this intentionally size `28`? Icons tend to be 16, 24, or 32px

Reply:
Updating